neo/Consensus/ConsensusService.cs

```diff
@@ -175,8 +175,21 @@ private void InitializeConsensus(byte viewNumber)
 
         private void Log(string message, LogLevel level = LogLevel.Info)
         {
+            var currentColor = new ConsoleColorSet();
+
+            switch (level)
+            {
+                case LogLevel.Debug: ConsoleColorSet.Debug.Apply(); break;
+                case LogLevel.Error: ConsoleColorSet.Error.Apply(); break;
+                case LogLevel.Fatal: ConsoleColorSet.Fatal.Apply(); break;
+                case LogLevel.Info: ConsoleColorSet.Info.Apply(); break;
+                case LogLevel.Warning: ConsoleColorSet.Warning.Apply(); break;
+            }
+
             Console.WriteLine($"[{DateTime.Now.TimeOfDay:hh\\:mm\\:ss\\.fff}] {message}");
             Plugin.Log(nameof(ConsensusService), level, message);
+
+            currentColor.Apply();
```

Comment:
@shargon, this way all the time we are going to set the log? In every message?

Reply:
We need to back to the previous color

Reply:
Onh man...But there is an other solution @igormcoelho

Reply:
This is good Vitor... it's just stream set/unset, it's bullet fast.